Sources/SwiftProtobuf/FieldNameMap.swift

```diff
@@ -24,62 +24,30 @@
 private let i_2166136261 = Int(bitPattern: 2166136261)
 private let i_16777619 = Int(16777619)
 
-private enum AsciiFieldName: Hashable {
-    case array([UInt8])
-    case mem(UnsafeBufferPointer<UInt8>)
+private struct AsciiFieldName: Hashable {
```

Comment:
We could just call this `ASCIIName` based on your comment elsewhere that it could be used for enum names too.